internal/commands/commands.go

```diff
@@ -32,6 +32,13 @@ type PackClient interface {
 	InspectExtension(client.InspectExtensionOptions) (*client.ExtensionInfo, error)
 	PullBuildpack(context.Context, client.PullBuildpackOptions) error
 	DownloadSBOM(name string, options client.DownloadSBOMOptions) error
+	CreateManifest(ctx context.Context, opts client.CreateManifestOptions) error
+	AnnotateManifest(ctx context.Context, opts client.ManifestAnnotateOptions) error
+	AddManifest(ctx context.Context, opts client.ManifestAddOptions) error
+	DeleteManifest(ctx context.Context, name []string) []error
```

Comment:
Also, I think returning the error array makes things difficult? maybe we concatenate the errors internally and just return one?